docs/add-ins/delegate-access.md

```diff
@@ -0,0 +1,94 @@
+
+## What is delegate access?
+
+Consider a business setting where an executive enables their assistant to manage the executive's mail and calendar. Depending on the level of access, the assistant or delegate can respond on the mailbox owner's behalf and create new appointments, among other tasks. To learn more about delegate access and how to give delegates permission, see [Allow someone else to manage your mail and calendar](https://support.office.com/article/allow-someone-else-to-manage-your-mail-and-calendar-41c40c04-3bd1-4d22-963a-28eafec25926).
+
+## Supported permissions
+
+You should know which delegate permissions the JavaScript for Office API supports.
+
+**Table 1 Supported permissions for delegate access**
+
+|Permission|Value|Description|Bit position<br>from the right|
+|---|:---:|---|:---:|
+|Read|1|Can read items.|1|
+|Write|2|Can create items.|2|
+|DeleteOwn|4|Can delete only the items they created.|3|
+|DeleteAll|8|Can delete any items.|4|
+|EditOwn|16|Can edit only the items they created.|5|
+|EditAll|32|Can edit any items.|6|
+
+> [!NOTE]
+> Using the API, you can only get existing delegate permissions.
+
+Also, the [DelegatePermissions](/javascript/api/outlook/office.mailboxenums.delegatepermissions) object is implemented using a bit mask to indicate all the delegate's permissions. This means that each position represents a particular permission and if it is set to `1` (i.e., it's on) then the delegate has the respective permission. For example, if the second bit from the right is `1`, then the delegate has **Write** permission. You can see an example of how to check for a specific permission in the [Use in messages and appointments for compose and read modes](#use-in-messages-and-appointments-for-compose-and-read-modes) section later in this article.
+
+## Configure the manifest
+
+You should set the [SupportsSharedFolders](/office/dev/add-ins/reference/manifest/supportssharedfolders) element to `true` under the parent element `DesktopFormFactor` so that your add-in is available in delegate access scenarios.
+
+> [!IMPORTANT]
+> The `SupportsSharedFolders` element is only available in the [Outlook add-ins preview requirement set](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview) against Exchange Online.
+>
+> Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.
+
+The following example shows the `SupportsSharedFolders` element set to `true` in a section of the manifest.
+
+```XML
+<DesktopFormFactor>
+  <FunctionFile resid="residDesktopFuncUrl" />
+  <SupportsSharedFolders>true</SupportsSharedFolders>
+  <ExtensionPoint xsi:type="PrimaryCommandSurface">
+    <!-- information about this extension point -->
+  </ExtensionPoint>
+</DesktopFormFactor>
+```
+
+## Use in messages and appointments for compose and read modes
```

Comment:
Suggest re-titling this section -> Get delegate permissions

Rationale behind this suggested revision:
- section title should describe the task that the section addresses (i.e., "what" the reader is learning how to do -- not "how" that task is done)

If the information that you currently have in the section title (i.e., "Use in messages and appointments for compose and read modes") is important for the reader to know, then you should incorporate it into the paragraph itself. For example, perhaps by making the first sentence read:
-> 
In message compose, message read, appointment compose, or appointment read mode, you can get the item's shared properties by calling the [item.getSharedPropertiesAsync](/office/dev/add-ins/reference/objectmodel/preview-requirement-set/office.context.mailbox.item#getsharedpropertiesasyncoptions-callback) method.